qiita_db/support_files/populate_test_db.sql

```diff
@@ -11,7 +11,7 @@ INSERT INTO qiita.qiita_user (email, user_level_id, password, name,
     '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe', 'Dude',
     'Nowhere University', '123 fake st, Apt 0, Faketown, CO 80302',
     '111-222-3344'),
-    ('shared@foo.bar', 3,
+    ('shared@foo.bar', 4,
```

Comment:
level 3 is superuser and 4 is use, shared user should be regular user so we can actually test that sharing/etc works ... let's see which other tests will fail ...